### PR TITLE
Get response content docs

### PR DESCRIPTION
**Pull request by KenitoInc**
- [x] Add Query Responses
- [x] Add more content to `Data Modification Response`

### CODE REVIEW

File: Odata-docs/client/v7/get-response-content.md [hunk 1]
```diff
@@ -32,3 +39,34 @@ static void Main(string[] args)
     }
 }
 ```
+The `DataServiceResponse` has the following properties that enable you to access additional information about a modification response:
+
+- [BatchHeaders](/dotnet/api/microsoft.odata.client.dataserviceresponse.batchheaders) - contains headers from an HTTP response associated with a batch request.
+- [BatchStatusCode](/dotnet/api/microsoft.odata.client.dataserviceresponse.batchstatuscode)- contains status code from an HTTP response associated with a batch request.
+- [IsBatchResponse](/dotnet/api/microsoft.odata.client.dataserviceresponse.isbatchresponse) - gets a Boolean value that indicates whether the response contains multiple results.
+
+The `DataServiceResponse` is enumerated to retrieve responses to operations being tracked by [OperationResponse](/dotnet/api/microsoft.odata.client.operationresponse) objects within the `DataServiceResponse`.
```
Comment (habbes) [inline Odata-docs/client/v7/get-response-content.md:48]: I find this a bit confusing. Maybe an example would help.

File: Odata-docs/client/v7/get-response-content.md [hunk 2]
```diff
@@ -32,3 +39,34 @@ static void Main(string[] args)
...
+
+## Query Responses
+We get a `Data Query Response` after calling `Execute()` or `ExecuteAsync()` on `DataServiceQueries`.
```
Comment (habbes) [inline Odata-docs/client/v7/get-response-content.md:51]: Is `Data Query Response` the same thing as `QueryOperationResponse`?

Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md:51]: `QueryOperationResponse` is the technical term. 
`Data Query Response` is the verbose term, meaning that `QueryOperationResponse` is a response from a Query request.

Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md:51]: I have changed to `QueryOperationResponse` to eliminate confusion.

File: Odata-docs/client/v7/get-response-content.md [hunk 3]
```diff
@@ -8,15 +8,22 @@ ms.date: 7/1/2019
...
  
 ---
-# Data modification response
+# Get Response Content
 **Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]
 
+## Data modification response
+We get a `Data Modification Response` after calling cache updating functions like `UpdateObject` and then `SaveChanges()` or  `SaveChangesAsync()`.
+
+Changes are tracked in the [DataServiceContext](/dotnet/api/microsoft.odata.client.dataservicecontext) instance but not sent to the server immediately. After you are finished with the required changes for a specified activity, call [SaveChanges](/dotnet/api/microsoft.odata.client.dataservicecontext.savechanges) to submit all the changes to the data service. `SaveChanges` will send a either a "POST", “PUT”, "PATCH" or “DELETE” request.
```
[the remark this answers is not in the document]
Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md:17]: Depends on the `SaveChangesOptions` you pass in the `SaveChanges()` method. https://docs.microsoft.com/en-us/dotnet/api/microsoft.odata.client.savechangesoptions

File: Odata-docs/client/v7/get-response-content.md (hunk 1 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md:48]: The example demonstrates that

Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md:48]:
```csharp
    DataServiceResponse responses = context.SaveChanges();

    foreach (var response in responses)
    {
        var changeResponse = (ChangeOperationResponse) response;
        var entityDescriptor = (EntityDescriptor) changeResponse.Descriptor;
        var personCreated = (Person) entityDescriptor.Entity; // the person created on the service
    }
```

File: Odata-docs/client/v7/get-response-content.md [hunk 4]
```diff
@@ -8,15 +8,22 @@ ms.date: 7/1/2019
 ms.topic: article
  
 ---
-# Data modification response
+# Get Response Content
 **Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]
```
Comment (mumbi-o) [inline Odata-docs/client/v7/get-response-content.md:13]:
```suggestion
In the previous section, we looked at how to carry out basic CRUD operations using the OData Client. In this section, we'll go ahead and drill a little deeper into how to handle the response object for each of these operations. We'll start with data modification responses for create, update and delete operations. We'll also look at query operations for read operations.
```

Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md:13]: #Resolved

File: Odata-docs/client/v7/get-response-content.md (hunk 2 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md]: #Resolved

File: Odata-docs/client/v7/get-response-content.md (hunk 1 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md:55]: #Resolved

File: Odata-docs/client/v7/get-response-content.md [hunk 5]
```diff
@@ -8,27 +8,74 @@ ms.date: 7/1/2019
...
 
+In the previous section, we looked at how to carry out basic CRUD operations using the OData Client. In this section, we'll go ahead and drill a little deeper into how to handle the response object for each of these operations. We'll start with data modification responses for create, update and delete operations. We'll also look at query operations for read operations.
```
Comment (gathogojr) [inline Odata-docs/client/v7/get-response-content.md:14]: Quoting previous section here presumes that the user is covering the content page by page. I believe the better way to refer to another page is by using a link. If someone without being aware of the "previous section" reference reorganizes the content, that will invalidate the reference. Plus you can't really refer to this as a "section", it's a "page" or "document"
E.g.
On [this](/odata/client/link) page, we looked at how to carry out basic CRUD operations using the OData Client. In this page, we'll go ahead and drill a little deeper into how to handle the response object for each of these operations.

or

In this page, we'll look at how to handle the responses from [basic CRUD operations](/odata/client/link).

File: Odata-docs/client/v7/get-response-content.md [hunk 6]
```diff
@@ -8,27 +8,74 @@ ms.date: 7/1/2019
...
+The `DataServiceResponse` has the following properties that enable you to access additional information about a modification response:
+
+- [BatchHeaders](/dotnet/api/microsoft.odata.client.dataserviceresponse.batchheaders) - contains headers from an HTTP response associated with a batch request.
+- [BatchStatusCode](/dotnet/api/microsoft.odata.client.dataserviceresponse.batchstatuscode)- contains status code from an HTTP response associated with a batch request.
+- [IsBatchResponse](/dotnet/api/microsoft.odata.client.dataserviceresponse.isbatchresponse) - gets a Boolean value that indicates whether the response contains multiple results.
+
+The `DataServiceResponse` is enumerated to retrieve responses to operations being tracked by [OperationResponse](/dotnet/api/microsoft.odata.client.operationresponse) objects within the `DataServiceResponse`.
+
+## Query Responses
+We get a [QueryOperationResponse](/dotnet/api/microsoft.odata.client.operationresponse) after calling `Execute()` or `ExecuteAsync()` on `DataServiceQueries`.
```
Comment (gathogojr) [inline Odata-docs/client/v7/get-response-content.md:60]:
```suggestion
We get a [QueryOperationResponse](/dotnet/api/microsoft.odata.client.operationresponse) after calling `Execute()` or `ExecuteAsync()` on `DataServiceQuery`.
```

Comment (gathogojr) [reply Odata-docs/client/v7/get-response-content.md:60]: If you apply code formatting, then it should be `DataServiceQuery` or `DataServiceQuery<T>` not `DataServiceQueries`

File: Odata-docs/client/v7/get-response-content.md [hunk 7]
```diff
@@ -8,27 +8,74 @@ ms.date: 7/1/2019
...
+Calling `Execute()` will send a "GET" request.
+
+When executed, the [DataServiceQuery&lt;TElement&gt;](/dotnet/api/microsoft.odata.client.dataservicequery-1) returns an `IEnumerable<T>` of the requested entity type. This query result can be cast to a [QueryOperationResponse&lt;T&gt;](/dotnet/api/microsoft.odata.client.queryoperationresponse-1)object, as in the following example:
```
Comment (gathogojr) [inline Odata-docs/client/v7/get-response-content.md:63]:
```suggestion
When executed, the [DataServiceQuery&lt;TElement&gt;](/dotnet/api/microsoft.odata.client.dataservicequery-1) returns an `IEnumerable<T>` of the requested entity type. This query result can be cast to a [QueryOperationResponse&lt;T&gt;](/dotnet/api/microsoft.odata.client.queryoperationresponse-1) object, as in the following example:
```

Comment (gathogojr) [reply Odata-docs/client/v7/get-response-content.md:63]: ... missing space the word "object"

File: Odata-docs/client/v7/get-response-content.md (hunk 6 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md:60]: Replaced it with `DataServiceQuery<TElement>`

File: Odata-docs/client/v7/get-response-content.md (hunk 7 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md]: #Resolved

File: Odata-docs/client/v7/get-response-content.md (hunk 6 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md]: #Resolved

File: Odata-docs/client/v7/get-response-content.md (hunk 5 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/get-response-content.md]: #Resolved

**Comment by gathogojr** [comment]
Same argument I made on your other PR https://github.com/MicrosoftDocs/OData-docs/pull/149#issuecomment-709480419
image
I think changing the link to `odata/client/v7/basic-crud-operations` would fix the warning

**Comment by KenitoInc** [comment]
> Same argument I made on your other PR [#149 (comment)](https://github.com/MicrosoftDocs/OData-docs/pull/149#issuecomment-709480419)
> image
> I think changing the link to `odata/client/v7/basic-crud-operations` would fix the warning

#resolved

**Review by gathogojr — APPROVED** [review]

**Pull request merged**